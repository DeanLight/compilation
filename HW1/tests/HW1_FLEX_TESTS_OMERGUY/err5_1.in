% this file is rated PG-13
( Hello my name is mighty \kiwi)