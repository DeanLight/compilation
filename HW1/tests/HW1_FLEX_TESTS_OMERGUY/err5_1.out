1 COMMENT % this file is rated PG-13
Error undefined escape sequence k
